templates/zerver/landing_nav.html

```diff
@@ -16,26 +16,89 @@
+                    <li>
+                        <a href="/why-zulip">Why Zulip</a>
+                    </li>
+                    <li>
+                        <a href="/features/">Features</a>
+                    </li>
+                    <li>
+                        <a href="/security/">Security</a>
+                    </li>
+                    <li>
+                        <a href="/integrations/">Integrations</a>
+                    </li>
+                    <li>
+                        <a href="/api/">REST API</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="dropdown nav-dropdown">
+                <div class="dropdown-label">Solutions<i class="fa fa-chevron-down"></i></div>
+                <ul class="dropdown-list solutions-options">
+                    <li>
+                        <a href="/for/companies/">Companies</a>
+                    </li>
+                    <li>
+                        <a href="/for/open-source/">Open source projects</a>
+                    </li>
+                    <li>
+                        <a href="/for/working-groups-and-communities/">Working groups and communities</a>
+                    </li>
+                </ul>
+            </div>
             <li data-on-page="plans">
-                <a href="/plans/">Plans</a>
+                <a href="/plans/">Pricing</a>
             </li>
+            <div class="dropdown nav-dropdown">
+                <div class="dropdown-label">Resources<i class="fa fa-chevron-down"></i></div>
+                <ul class="dropdown-list resources-options">
+                    <div class="dropdown-column">
+                        <li class="nav-header">For users</li>
+                        <li>
+                            <a href="/help/getting-started-with-zulip">Getting started</a>
+                        </li>
+                        <li>
+                            <a href="/help">Help center</a>
+                        </li>
+                        <li>
+                            <a href="https://zulip.readthedocs.io/en/latest/contributing/chat-zulip-org.html" target=”_blank”>
+                                Community chat
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/help/contact-support">Contact support</a>
+                        </li>
+                        <li>
+                            <a href="https://blog.zulip.com/">Blog</a>
+                        </li>
+                    </div>
+                    <div class="dropdown-column">
+                        <li class="nav-header">For administrators</li>
+                        <li>
+                            <a href="https://zulip.readthedocs.io/en/stable/production/index.html" target=”_blank”>
+                                Installing a Zulip server
+                            </a>
+                        </li>
```

Comment:
Do we want "Upgrade a Zulip server" to be right after the installation docs?  I think it is probably a nice touch to do so, especially since the after sysadmin should upgrade more often than install.

Reply:
I like it!